index.html

```
@@ -2777,5 +2777,87 @@ <h2>Keyboard Shortcuts</h2>
+    } else {
+      btn.style.display = "none";
+    }
+  });
+
+  btn.addEventListener("click", () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth"
+    });
+  });
+</script>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </body></html> ], open tag match failed [ <body class="bg-background text-on-surface font-body antialiased"> ] on line 440.

(tag-pair)